django/thunderstore/repository/models/package_version.py

```diff
@@ -120,6 +140,13 @@ class PackageVersion(VisibilityMixin, AdminLinkMixin):
     readme = models.TextField()
     changelog = models.TextField(blank=True, null=True)
 
+    # TODO: Default should be pending once all versions require automated scanning before appearing to users
+    review_status = models.CharField(
```

Comment:
Use `TextField` instead of `CharField`; they're the same underlying field in postgres but `CharField` has unnecessary constraints around max length (unlikely to be hit, but there's no reason to not use `TextField` here so it should be the go-to choice)